react/README.md

```diff
@@ -50,6 +59,19 @@ The following table shows the relationships between the application type attribu
 | Web	| authorization_code	| code	| client_secret_basic| 
 | Single-page	| implicit	| token, id_token	| none| 
 
+## External Identity Providers 
+
+An identity provider configuration allows linked users to authenticate and gain access to PingOne resources using the login flow and credentials provided by the external identity provider.
+For now PingOne supports Facebook and SAML as external identity providers.
+
+### Social Login Configuration
+
+The PingOne identity provider resource representing an external provider includes mapping attributes that reference the user attribute (or attributes) from the external identity provider that are mapped to PingOne user attributes.
+To add Facebook as the the external identity provider, please follow this steps:
+- create a new connection under ***Identity Providers**  part n PingOne admin console (check [Identity providers](https://apidocs.pingidentity.com/pingone/customer/v1/api/man/p1_IdentityProvider/) for more information)
+- in PingOne admin console, select your Sign-on policy, click edit and under **Enabled Identity Providers** select the previously added connection
+- make sure that external identity provider will accept PingOne domain redirect url. For the Facebook Login, please set under **Client OAuth Settings** URL with a following pattern: `https://auth.pingone.com/c2c2b4f8-c3da-4b23-abef-457ceaf25591/rp/callback/facebook` under **Valid OAuth Redirect URIs** 
```

Comment:
they need to insert their environment id right?

Reply:
Ah, missed that hardcoding. Thanks, fixed it